### PR TITLE
Add version and help arguments

### PR DESCRIPTION
**Pull request by calebhearth**
`-v / --version` will print version information and exit.
`-h / --help` will print help information and exit.

Any other flag will complain and exit with a failure code.

### CODE REVIEW

File: main.go [hunk 1]
```diff
@@ -17,3 +29,19 @@ func main() {
 	t := tree.Tree(lines, "/")
 	t.Print(0, nil)
 }
+
+func handleArguments() {
```
Comment (bernerdschaefer) [inline main.go:33]: Did you consider using the `flag` package for this?

Comment (calebhearth) [inline main.go:33]: I did, and ended up removing it.
1. Flag doesn't handle `--` for whatever reason.
2. https://github.com/jessevdk/go-flags felt heavy-handed for this.

thoughts?

Comment (bernerdschaefer) [inline main.go:33]: Oh, I see. Yeah, it does handle `--`, but indeed does not have a concept of short vs. long flags.

File: main.go [hunk 2]
```diff
@@ -17,3 +29,19 @@ func main() {
...
+	for _, a := range os.Args[1:] {
+		switch a {
+		case "--version", "-v":
+			fmt.Printf("ftree %s\n", version)
+			os.Exit(0)
+		case "--help", "-h":
+			fmt.Print(help)
+			os.Exit(0)
+		default:
+			fmt.Printf("No such argument: %v\n", a)
```
Comment (bernerdschaefer) [inline main.go:43]: Perhaps `fmt.Fpritnf(os.Stderr, ...)`?

Comment (calebhearth) [inline main.go:43]: Yeah good call

**Pull request closed without merging**